ephios/core/templates/core/userprofile_list.html

```diff
@@ -1,64 +1,101 @@
+                                {% if not userprofile.is_active %}
+                                    <span class="badge bg-danger">{% translate "Inactive" %}</span>
+                                {% elif userprofile.is_staff %}
+                                    <span class="badge bg-info">{% translate "Admin" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-8 col-lg">
+                        <div>
+                            {% if userprofile.groups.all %}
+                                <span class="fas fa-users"></span>
+                                {{ userprofile.groups.all|join:", " }}
+                            {% else %}
+                                <span class="fas fa-users-slash" title="{% translate "no groups" %}"></span>
+                            {% endif %}
+                        </div>
+                        <div class="text-body-secondary">
+                            {% for abbreviation in userprofile.qualification_grants.all|grants_to_essential_abbreviations %}
+                                <span class='badge bg-secondary text-dark'>{{ abbreviation }}</span>
+                            {% endfor %}
+                        </div>
+                    </div>
+                    <div class="col-4 col-lg-auto d-flex align-items-center justify-content-end">
                         <a class="btn btn-secondary btn-sm text-nowrap"
-                               href="{% url "core:workinghours_detail" userprofile.pk %}"><span
-                                    class="fa fa-eye"></span> <span
-                                    class="d-none d-md-inline">{% trans "View working hours" %}</span></a>
+                           href="{% url "core:workinghours_detail" userprofile.pk %}"><span
+                                class="fa fa-eye"></span> <span
+                                class="d-none d-md-inline">{% translate "Working hours" %}</span></a>
                         <a class="btn btn-secondary btn-sm text-nowrap ms-1"
                            href="{% url "core:userprofile_edit" userprofile.pk %}"><span
                                 class="fa fa-edit"></span> <span
-                                class="d-none d-md-inline">{% trans "Edit" %}</span></a>
+                                class="d-none d-md-inline">{% translate "Edit" %}</span></a>
                         {% if perms.core.delete_userprofile %}
                             <a class="btn btn-secondary btn-sm text-nowrap ms-1"
                                href="{% url "core:userprofile_delete" userprofile.pk %}"><span
                                     class="fa fa-trash-alt"></span> <span
-                                    class="d-none d-md-inline">{% trans "Delete" %}</span></a>
+                                    class="visually-hidden">{% translate "Delete" %}</span></a>
```

Comment:
I think its odd that the poor trashcan has no label

Reply:
they got it back :taco: